### DISCUSSION TITLE
[source-postgres] Do the connector sync Indexes?

### DISCUSSION THREAD

**Discussion by agalewsky-pnn** [airbytehq/airbyte#34361] [Connector Questions] [answered]
I have searched and searched but I can’t find if I am going to from one Postgres to another Postgres if the indexes get replicated as well.  They don’t seem to be.

**Comment by marcosmarxm** [comment, marked as answer]
@agalewsky-pnn Airbyte won't sync indexes and other data objects (functions, indexes, permissions) to the destination. 
(I think and probably need confirmation from engineering team) this is due each database has different ways to manage these objects and tables and views there is an standard method to handle them.
I made this pull request to improve the current documentation https://github.com/airbytehq/airbyte/pull/34362

**Reply by Doc-Savage** [reply]
Thanks for the quick clarification.  What is considered best practices to take care of this.  I would argue that the replication is almost useless without the indexes.

**Reply by marcosmarxm** [reply]
Airbyte is a data integration tool where you can sync data from any source to any destination. If you need a replication tool to migrate a database to another same database (at the moment) Airbyte won't best suit your requirements.

[a comment by an automated account is not shown]
